condominium/data/ir_model_fields.xml

```diff
@@ -1,577 +1,1674 @@
+    <field name="field_description">Owner</field>
+    <field name="model_id" ref="model_x_property_line_owner"/>
+    <field name="name">x_owner</field>
+    <field name="relation">res.partner</field>
+  </record>
+  <record id="field_analytic_owner" model="ir.model.fields">
+    <field name="ttype">many2one</field>
+    <field name="copied" eval="True"/>
+    <field name="field_description">Owner</field>
+    <field name="model_id" ref="analytic.model_account_analytic_account"/>
+    <field name="name">x_owner</field>
+    <field name="relation">res.partner</field>
+  </record>
+  <record id="field_analytic_property" model="ir.model.fields">
+    <field name="ttype">many2one</field>
+    <field name="copied" eval="True"/>
+    <field name="field_description">Property</field>
+    <field name="model_id" ref="analytic.model_account_analytic_account"/>
+    <field name="name">x_property</field>
+    <field name="relation">x_property</field>
+  </record>
+  <record id="field_property_line_account" model="ir.model.fields">
+    <field name="ttype">many2one</field>
+    <field name="copied" eval="True"/>
+    <field name="field_description">Account</field>
+    <field name="model_id" ref="model_x_property_line_owner"/>
+    <field name="name">x_account</field>
+    <field name="relation">account.analytic.account</field>
+  </record>
+  <record id="field_analytic_owner_line" model="ir.model.fields">
+    <field name="ttype">many2one</field>
+    <field name="copied" eval="True"/>
+    <field name="field_description">Owner line</field>
+    <field name="model_id" ref="analytic.model_account_analytic_account"/>
+    <field name="name">x_owner_line</field>
+    <field name="relation">x_property_line_owner</field>
+  </record>
+  <record id="field_account_analytic_account_start_date" model="ir.model.fields">
+    <field name="ttype">date</field>
+    <field name="related">x_owner_line.x_start_date</field>
+    <field name="field_description">Start Date</field>
+    <field name="model_id" ref="analytic.model_account_analytic_account"/>
+    <field name="name">x_start_date</field>
+    <field name="readonly" eval="True"/>
+    <field name="store" eval="False"/>
+  </record>
+  <record id="field_account_analytic_account_end_date" model="ir.model.fields">
+    <field name="ttype">date</field>
+    <field name="related">x_owner_line.x_end_date</field>
+    <field name="field_description">End Date</field>
+    <field name="model_id" ref="analytic.model_account_analytic_account"/>
+    <field name="name">x_end_date</field>
+    <field name="readonly" eval="True"/>
+    <field name="store" eval="False"/>
+  </record>
+  <record id="field_ratios_account" model="ir.model.fields">
+    <field name="compute"><![CDATA[for record in self:
+    accounts = self.env['account.analytic.account'].search([('partner_id','=',record.x_owner),('x_property','=',record.x_property),('x_end_date','=',False)])
+    record['x_account'] = accounts.id]]></field>
```

Comment:
what if search returns multiple analytic accounts? accounts.id will break.
if you want only one analytic account, use the limit=1 in search and change the name account from accounts.
if expecting multiple, then handle the assignation proper.